### DISCUSSION TITLE
Some general questions about Lie groups.

### DISCUSSION THREAD

**Discussion by Dhanushvarma** [facebookresearch/theseus#667] [Q&A] [answered]
- Let's say we have to SE3, named A and B, if we perform `A.compose(B)` What is the resulting homogeneous transformation, I want clarity on the order of the multiplication.

- When we do `A.local(B)` What exactly is being computed for the above mentioned SE3, if you could point to resources that explain this, it would be greatly appreciated

**Comment by fantaosha** [comment, marked as answer]
Hello, the answers are the follows.

- Suppose `A=(R0, t0)`, `B=(R1, t1)`. Then `A.compose(B)` is essentially `(R0@R1, R0@t1 + t0)`
- `A.local(B)` is `log(A^-1@B)`

Thanks!
